protocol-designer/src/localization/en/modal.json

```
@@ -42,6 +42,14 @@
     "deckConfigAnd96Channel": {
       "body1": "Introducing the {{pd}} 8.0 with deck configuration and 96-channel pipette support!",
       "body2": "All protocols now require {{app}} version <strong> 7.1+ </strong> to run."
+    },
+    "customParamsAndMultiTipAndModule": {
+      "body1": "Introducing the {{pd}} 8.1. Starting today, you will be able to:",
+      "body2": "Customize blowout flowRate speed and height",
+      "body3": "Customize aspirate, dispense, and mix <i>'x'</i> and <i>'y'</i> well positioning",
+      "body4": "Assign up to 3 types of tipracks to a single pipette",
+      "body5": "For the Flex only; add up to 7 Temperature Modules on the deck at the same time",
+      "body6": "All protocols require {{app}} version <strong> 7.2+ </strong> to run."
```

can you review the copy here when you have a minute @ecormany?

---

```suggestion
      "body1": "Introducing {{pd}} 8.1. Starting today, you will be able to:",
      "body2": "Customize blowout speed and height.",
      "body3": "Adjust horizontal position within a well when aspirating, dispensing, or mixing.",
      "body4": "Assign up to three types of tip racks to a single pipette.",
      "body5": "Add multiple Temperature Modules to the deck (Flex only).",
      "body6": "All protocols require {{app}} version <strong>7.2.0 or later</strong> to run."
```
Double-check me: is it 7.2.0 for the required robot software?

---

There are pipette collision logic updates that were added in App 7.2.0 that will be mirrored in PD 8.1. Otherwise, all the added functionality in this release has been supported in App prior to App 7.2.0